fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -1564,15 +1564,45 @@ class ShardManager {
 			}
 		}
 
-		auto currentCfOptions = active ? rState->getCFOptions() : rState->getCFOptionsForInactiveShard();
-		auto [it, inserted] = physicalShards.emplace(id, std::make_shared<PhysicalShard>(db, id, currentCfOptions));
-		std::shared_ptr<PhysicalShard>& shard = it->second;
+		auto it = physicalShards.find(id);
+		std::shared_ptr<PhysicalShard> physicalShard = nullptr;
+		if (it != physicalShards.end()) {
+			physicalShard = it->second;
+			// TODO: consider coalescing continous key ranges.
+			if (!SERVER_KNOBS->SHARDED_ROCKSDB_ALLOW_MULTIPLE_RANGES) {
+				bool continous = physicalShard->dataShards.empty();
+				std::string rangeStr = "";
+				for (auto& [_, shard] : physicalShard->dataShards) {
+					rangeStr += shard->range.toString() + ", ";
```

Comment:
We add a ', ' here as though we were going to add something else to the string but we don't add anything more in this code block. Is the ', ' intentional?  Perhaps it is meant to convey something? (rangeStr is safe to output in the trace? Does it have binary in it or if it does, it is escaped so can be printed)?

Reply:
`toString()` takes care of escape characters. ', ' is added to make is more readable. The trace event will be valid.